test/common/http/access_log/access_log_impl_test.cc

```diff
@@ -108,7 +109,7 @@ TEST_F(AccessLogImplTest, LogMoreData) {
   request_headers_.addViaCopy(Http::Headers::get().ForwardedFor, "x.x.x.x");
 
   log->log(&request_headers_, &response_headers_, request_info_);
-  EXPECT_EQ("[1900-01-01T00:00:00.000Z] \"GET / HTTP/1.1\" 0 UF 1 2 3 - \"x.x.x.x\" "
+  EXPECT_EQ("[1999-01-01T00:00:00.000Z] \"GET / HTTP/1.1\" 0 UF 1 2 3 - \"x.x.x.x\" "
```

Comment:
Can you including a reference to this bug's details in your commit message? Seems surprising OS X isn't able to work on dates in 1900. OTOH, there was no HTTP then :)